pkg/sidecar-controller/groupsnapshot_helper.go

```diff
@@ -251,6 +251,10 @@ func (ctrl *csiSnapshotSideCarController) deleteCSIGroupSnapshotOperation(groupS
 		for _, contentRef := range groupSnapshotContent.Status.PVVolumeSnapshotContentList {
 			snapshotContent, err := ctrl.contentLister.Get(contentRef.VolumeSnapshotContentRef.Name)
 			if err != nil {
+				if errors.IsNotFound(err) {
+					klog.Infof("snapshot content %v not found", contentRef.VolumeSnapshotContentRef.Name)
```

Comment:
This should be a warning. If snapshot content not found, that means it is somehow forcefully deleted. A snapshot content can't be deleted if group snapshot handle is still in its status.
Can you tell me how you reproduced this problem?

Reply:
yes even I thought the same, but this issue was reproducible when we repeatedly create and delete a VGS. As per my debug and testing, since the creation and deletion was taking place with same name within seconds, it seems there was a sync issue due to which the snapshot content got deleted . There is a script that we used to reproduce this issue, let me know if you want I can share it with you.

Reply:
Hey, I think we dont need this check anymore, it seems the VGS is getting successfully deleted.